### ISSUE TITLE
AttributeError on Classified Polygon People

### ISSUE THREAD

**Issue by ismailsunni** [inasafe/inasafe#2853]
### Problem
I was trying to test the issue number 2 from #2846, intead got TypeError, I got AttributeError like below:

```
Processing started

Please wait - processing may take a while depending on your hardware configuration and the analysis extents and data.

This analysis will calculate the impact of Flowdepth Level 5 Vector on Populasi and bullet list the results aggregated by Desa .

Select attribute

Please select which attribute you want to use as ID for the aggregated results

Preparing aggregation layer

clipping the aggregation layer to match the intersection of the hazard and exposure layer extents.

Select attribute

Please select which attribute you want to use as ID for the aggregated results

Calculating impact

This may take a little while - we are computing the areas that will be impacted by the hazard and writing the result to a new layer.

Problem

The following problem(s) were encountered whilst running the analysis.

AttributeError
Suggestion

You can try the following to resolve the issue:

Check that you have the latest version of InaSAFE installed - you may have encountered a bug that is fixed in a subsequent release.
Check the InaSAFE documentation to see if you are trying to do something unsupported.
Report the problem using the issue tracker at https://github.com/AIFDR/inasafe/issues. Reporting an issue requires that you first create a free account at http://github.com. When you report the issue, please copy and paste the complete contents of this panel into the issue to ensure the best possible chance of getting your issue resolved.
Try contacting one of the InaSAFE development team by sending an email to info@inasafe.org. Please ensure that you copy and paste the complete contents of this panel into the email.
Details

These additional details were reported when the problem occurred.

An exception occurred when running the impact analysis.
'NoneType' object has no attribute 'wkbType'
Diagnostics

In file "/Users/ismailsunni/.qgis2/python/plugins/inasafe-dev/safe/impact_functions/base.py", line 679, in run_analysis self._impact = self._calculate_impact()
In file "/Users/ismailsunni/.qgis2/python/plugins/inasafe-dev/safe/impact_functions/base.py", line 1411, in _calculate_impact result_layer = self.run()
In file "/Users/ismailsunni/.qgis2/python/plugins/inasafe-dev/safe/impact_functions/generic/classified_polygon_people/impact_function.py", line 163, in run impact_fields)
In file "/Users/ismailsunni/.qgis2/python/plugins/inasafe-dev/safe/impact_functions/generic/classified_polygon_people/impact_function.py", line 355, in evaluate_impact if not impact_geometry.wkbType() == QGis.WKBPolygon and \
```

### Expected Behaviour
IF running properly without error

### Environment
* InaSAFE : Develo (264e679)
* QGIS : 2.8.3
* OS : OSX
* Data used : See detailds on #2846 second issue.

### Proposed Solution
Make it happy

**Comment by ismailsunni** [comment]
This problem is also occurs when trying the third scenario in #2846, but without aggregation. For the aggregation, I got the same TypeError.

I have added the fix also in the #2854

**Comment by ismailsunni** [comment]
Fixed in #2854
